app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -33,10 +37,16 @@ def process_async_state(current_async_state)
         )
       end
```

Comment:
I *think* we need to make sure that this method (and by proxy `#call`) returns the `FormResponse` object so it gets logged correctly

Reply:
I wasn't super sure about this action, it doesn't have any specs, and it didn't seem to return a `FormResponse` before?

@aduth @stevegsa looks like y'all edited this recently, does this need to return a `FormResponse`?

Reply:
At first glance I'm not entirely sure, though I do know for most non-JSON-rendering steps, it's expected the class would implement a [`form_submit`](https://github.com/18F/identity-idp/blob/bc1a2240899b7400e9e460c5b1611c6133e32105/app/services/flow/base_step.rb#L10-L170) which is expected to return a `FormResponse`. Since we're rendering JSON, my understanding is that we're sort of bypassing that default behavior. Whether `#call` is also expected to return `FormResponse` is another question though, and I'd need to dig a little further to know for sure.

Reply:
`render_json` is side-effecty, it just sets `@json = ` and moves on, it does not need to return a particular value or be the last return in any of these.

[`BaseFlow#wrap_send`](https://github.com/18F/identity-idp/blob/master/app/services/flow/base_flow.rb#L58-L62) and [`BaseStep#base_call`](https://github.com/18F/identity-idp/blob/master/app/services/flow/base_step.rb#L10-L17) both will check if the return values are a `FormResponse` so that we can handle and log that appropriately

Reply:
Added in https://github.com/18F/identity-idp/pull/4366/commits/9cf5a593e8a8a24a0e50d7da5d6d996646e3d419